core/src/odbc_uri.rs

```diff
@@ -285,21 +285,26 @@ impl ODBCUri {
         }
         Self::set_server_and_source(&mut client_options, server, source.map(String::from))?;
         client_options.app_name = client_options.app_name.or(self.handle_app_name());
-        Ok(client_options)
+        let connection_string =
+            ConnectionString::parse(uri).map_err(Error::InvalidClientOptions)?;
+        Ok((client_options, Some(connection_string)))
```

Comment:
Do we know why the rust team decided not to include `uuidRepresentation` in the `ClientOptions` struct? Pretty unfortunate we need to change all of this to deal with a tuple of `(ClientOptions, ConnectionString)` essentially _just_ for the one option we use that isn't held in `ClientOptions`.

Reply:
Yeah. I actually did the work for this (adding it to the rust driver), and that's where they wanted it added. In the spec it's not a client option but a URI option, so that's what we have.